bin/compile

```diff
@@ -295,6 +295,12 @@ ensureGo() {
 
     export GOROOT="${goPath}"
     PATH="${goPath}/bin:${PATH}"
+
+    # Export GOCACHE if Go >= 1.10
+    local goCachePath=$(go env | grep GOCACHE)
```

Comment:
I think this could be: `if go env | egrep -q '^GOCACHE='; then`.

Reply:
TIL. My bash skill makes me mad 😝

Reply:
Fixed